install/image-references

```diff
@@ -26,3 +26,11 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-azure-machine-controllers:v4.0.0
+ - name: baremetal-operator
+    from:
+      kind: DockerImage
+      name: quay.io/metal3-io/baremetal-operator:latest
```

Comment:
This needs to change to refer to the official openshift image of baremetal-operator, which is already being built.  Then the configmap can refer to the openshift image, as well.

https://quay.io/repository/openshift/origin-baremetal-operator

Reply:
The official OpenShift images were added as part of https://github.com/openshift/machine-api-operator/pull/367.